app/javascript/routes/LeaseUpRoutes.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+// import ReactDOM from 'react-dom'
+import { Switch, Route } from 'react-router-dom'
+
+import LeaseUpApplicationsPage from '~/components/lease_ups/LeaseUpApplicationsPage'
+import SupplementalApplicationPage from '~/components/supplemental_application/SupplementalApplicationPage'
+import LeaseUpListingsPage from '~/components/lease_ups/LeaseUpListingsPage'
+import ApplicationPage from '~/components/applications/ApplicationPage'
+
+const LeaseUpRoutes = () => (
+  <Switch>
+    <Route exact path='/listings/lease-ups/:listingId/applications'>
+      <LeaseUpApplicationsPage />
+    </Route>
+    <Route exact path='/listings/lease-ups'>
+      <LeaseUpListingsPage />
+    </Route>
+    <Route
+      exact
+      path='/applications/:applicationId/supplementals'
+      render={({ match }) => (
+        // TODO: Convert SupplementalApplicationPage to functional components and use
+        // hooks to get path params.
+        <SupplementalApplicationPage applicationId={match.params.applicationId} />
+      )}
+    />
+    <Route exact path='/applications/:applicationId'>
```

Comment:
@akegan I haven't thought much about it, so thanks for bringing it up.

I think my next PR should be updating the paths, and I think that's a good time to split the pre-lottery section to use its own path.

How do you feel about still loading the pre-lottery apps via LeaseUpRoutes? It would be a separate route and path but would still load the ApplicationPage component. That would definitely be easiest from a technical perspective.

Maybe we re-name LeaseUpApp and LeaseUpRoutes to be more generic in that case

Reply:
I'm going to go ahead and merge, I'll make sure to handle this in the path/url update pr

Reply:
sure yeah that sounds reasonable.